src/simulator/base.py

```diff
@@ -0,0 +1,49 @@
+import object
+from position import Position
+from shape import Shape
+import object_registry
+import numpy as np
+
+RadioID = object.ObjectId
+
+
+class Base(object.Object):
+    """
+    A physical station, primarily in charge of communicating with robots
+    and facilitating their navigation by transmitting their (estimated) distances with the base.
+    """
+
+    def __init__(self, oid: object.ObjectId, name: str, shape: Shape, position: Position,
+                 owner_object: object.Object, registry: object_registry.ObjectRegistry,
+                 rid: RadioID, standard_deviation: float):
+        super().__init__(oid, name, shape, position, owner_object, registry)
+        self.radio_id = rid
+        self.standard_deviation = standard_deviation
+        self.__timer = 0
+
+    def get_distance(self, position: Position):
```

Comment:
return value type hinting

Reply:
We have two types for the return value. Add Return floating type hinting?

Reply:
good point. float is okay for now. But please remember to talk about it in the meeting.